### PR TITLE
add artifact creation worker

### PR DESCRIPTION
**Pull request by GregorioDiStefano**
Signed-off-by: Gregorio Di Stefano <greg.distefano+github@gmail.com>

### CODE REVIEW

**Comment by GregorioDiStefano** [comment]
There is a bug in mender-artifacts that prevents mender artifact modification from work, there is an open pr:  https://github.com/mendersoftware/mender-artifact/pull/100

will have to fix on Monday

**Comment by GregorioDiStefano** [comment]
@mendersoftware/rndity @maciejmrowiec

File: workers/artifact_creator/main.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+import json
+from conductor.ConductorWorker import ConductorWorker
+
+logging.basicConfig()
+log = logging.getLogger('conductor-workers.create_tenant_artifacts_images')
+log.setLevel(logging.INFO)
+
+conf = config.get_config()
+
+BUCKET_NAME = os.getenv("BUCKET")
```
Comment (mchalski) [inline workers/artifact_creator/main.py:20]: env is good, but mind you - configarparse (hidden by config.py) supports both cmd line args and envs. so each arg has an env var counterpart. so - use that instead?

Comment (mchalski) [reply workers/artifact_creator/main.py:20]: also these checks could then be easily part of check_config()

File: workers/artifact_creator/main.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+        return {'status': 'FAILED',
+                'output': err,
+                'logs': []}
+    
+    # cleanup temp folder
+    finally:
+        shutil.rmtree(tempfolder)
+
+def main():
+    log.info('create_tenant'.format(conf.conductor))
```
Comment (mchalski) [inline workers/artifact_creator/main.py:156]: create artifacts?

File: workers/artifact_creator/main.py [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    cc = ConductorWorker(conf.conductor + "/api", 1, 0.1)
+
+    check_config()
+    cc.start('create_tenant', create_artifacts, True)
```
Comment (mchalski) [inline workers/artifact_creator/main.py:160]: create_artifacts?

**Pull request closed without merging**